Gui/opensim/view/src/org/opensim/view/nodes/OneForceNode.java

```diff
@@ -55,9 +57,10 @@ public class OneForceNode extends DisablablModelComponentNode implements Colorab
      */
     public OneForceNode(OpenSimObject force) {
         super(force);
-        hasPath = force.hasProperty("GeometryPath");
+        Force typedForce = Force.safeDownCast(force);
+        hasPath = typedForce.hasProperty("path") && typedForce.hasVisualPath();
```

Comment:
Is it sufficient to just use `typedForce.hasVisualPath()` here?

Reply:
### Based on the comments in hasVisualPath I believe you're right. Thanks for the review @carmichaelong